package.json

```diff
@@ -34,12 +34,16 @@
 	"scripts": {
 		"build": "yarn clean && lerna run build --stream",
 		"clean": "lerna run clean",
-		"ganache-cli": "npx ganache-cli -m 'smart guide what forget tired jungle always expire rescue boring glue champion'",
+		"ganache-cli": "npx ganache-cli -m 'smart guide what forget tired jungle always expire rescue boring glue champion' --account=\"0x4c3758228f536f7a210f8936182fb5b728046970b8e3215d0b5cb4c4faae8a4e,100000000000000000000\" --account=\"0x34aeb1f338c17e6b440c189655c89fcef148893a24a7f15c0cb666d9cf5eacb3,100000000000000000000\"",
+		"ganache-cli-background": "yarn ganache-cli > /dev/null & client=$! npx wait-port 8545",
```

Comment:
Should be updated to use `ganache` instead of `ganache-cli`

Reply:
done